src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -121,7 +121,7 @@ internal void Invoke(ApplicationEngine engine)
                 throw new InvalidOperationException("It is not allowed to use Neo.Native.Call directly to call native contracts. System.Contract.Call should be used.");
             ExecutionContext context = engine.CurrentContext;
             string operation = context.EvaluationStack.Pop().GetString();
-            ContractMethodMetadata method = methods[operation];
+            ContractMethodMetadata method = methods[(operation, context.EvaluationStack.Count)];
```

Comment:
It could be possible to have more items before cal the method? 🤔

Reply:
It's impossible.

Reply:
`InvocationScript` it's controlled by the user, and maybe it can choose a different method if he add a new item.

Reply:
If he pushes more or less items, the contract should fail.